src/controllers/cohort.js

```diff
@@ -51,3 +52,37 @@ export const getCohortById = async (req, res) => {
     return sendMessageResponse(res, 500, 'Unable to get cohort')
   }
 }
+
+export const getAllCohortExercises = async (req, res) => {
+  const cohortId = Number(req.params.id)
+  const whereData = {}
+  whereData.cohortId = cohortId
+
+  try {
+    const foundExercise = await CohortExercise.findAll({ whereData })
```

Comment:
I expect variables containing arrays to be named using plural nouns - so `foundExercises`

Reply:
done